src/SPC/builder/extension/imagick.php

```diff
@@ -12,7 +12,7 @@ class imagick extends Extension
 {
     public function patchBeforeMake(): bool
     {
-        if (getenv('SPC_LIBC') !== 'musl') {
+        if (getenv('SPC_LIBC') === 'glibc' && str_contains(getenv('CC'), 'devtoolset-10')) {
```

Comment:
What is the reason for this

Reply:
Detecting if libgomp was built with -fPIC is very messy, short of linking against it with imagemagick source and running ldd, at which point it's already too late.

Reply:
I have two separate PR's that tackle this in a different way, though. Either building libgomp with the correct version or building our own gcc + binutils chain.